src/engine/instrument.h

```diff
@@ -367,14 +369,24 @@ struct DivInstrumentC64 {
 };
 
 struct DivInstrumentAmiga {
+  enum DivReverseMode: unsigned char {
```

Comment:
Question regarding reverse: Is it possible to use the DIV_CMD_SAMPLE_DIR approach? It's how we currently do it.

Reply:
I added this method: sample tab (currently only working on YM2612 and ES5506), effect command, and macro